conf/messages

```diff
@@ -173,12 +173,13 @@ ssttp.calculator.misalignment-page.subtitle=The figures you entered don’t matc
 ssttp.calculator.misalignment-page.total-entered=You entered:
 ssttp.calculator.misalignment-page.total-in-account=Your tax account shows:
 ssttp.calculator.misalignment-page.account-breakdown.title=View a breakdown
-ssttp.calculator.misalignment-page.account-breakdown.total=TOTAL:
+ssttp.calculator.misalignment-page.account-breakdown.total=Total:
 ssttp.calculator.misalignment-page.possible-reasons.title=These differences could be because:
-ssttp.calculator.misalignment-page.possible-reasons.list.1=you haven’t filed your Self Assessment tax return
-ssttp.calculator.misalignment-page.possible-reasons.list.2=you filed less than 3 days ago and your account hasn’t updated yet
-ssttp.calculator.misalignment-page.possible-reasons.list.3=you didn’t include all outstanding amounts in your calculation
-ssttp.calculator.misalignment-page.possible-reasons.list.4=you included an amount that isn't related to your Self Assessment
+ssttp.calculator.misalignment-page.possible-reasons.list.1=didn’t include all outstanding amounts in your calculation
+ssttp.calculator.misalignment-page.possible-reasons.list.2=included an amount that isn't related to your Self Assessment
```

Comment:
The apostrophe is missing on the page, need to double it '' to escape the apostrophe, or copy the one used in the line above